tools/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -68,6 +71,25 @@ object ToolUtils extends Logging {
     // floor is applied after multiplying by 100. This keeps the number "as is" up-to two decimal.
     math.floor(valNum * 100) / 100
   }
+
+  def renderTextField(values: Seq[Any], itemDelimiter: String, txtDelimiter: String) : String = {
+    replaceDelimiter(values.mkString(itemDelimiter), txtDelimiter)
+  }
+
+  def parseComplexTypes(
```

Comment:
Using the name parse here seems odd to me, perhaps formatComplexTypes or ComplexTypesToString.  Same with functions below

Reply:
Replaced "parse" with "format"